### ISSUE TITLE
SA1300 and SA1313 conflict on `record` declarations

### ISSUE THREAD

**Issue by tillig** [DotNetAnalyzers/StyleCopAnalyzers#3464]
Given this code:

```c#
public class Tests
{
  private record Component(IOtherService Service);
}
```

SA1300 and SA1313 conflict:

- SA1300: Element 'service' should begin with an uppercase letter.
- SA1313: Parameter 'Service' should begin with a lowercase letter.

Using .NET 6 with StyleCop.Analyzers 1.2.0-beta.406.

**Comment by sharwell** [comment]
> Using .NET 6 with StyleCop.Analyzers 1.2.0-beta.406.

Are you certain this is correct? I am not able to reproduce this, and it should be fixed in 1.2.0-beta.261 and newer (https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3181).

**Comment by tillig** [comment]
Hmmm. This may be an artifact of OmniSharp. After a full exit of VS Code, a `git clean -dfx`, and a restart... I'm _also_ not able to replicate it. Bah. Sorry for the trouble.
